### DISCUSSION TITLE
Вопрос по небольшим смысловым блокам

### DISCUSSION THREAD

**Discussion by vitalybaev** [doka-guide/content#2663] [Q&A] [answered]
Привет!

У меня тут появился вопрос - периодически будут возникать небольшие кусочки информации, которые будут дублироваться в разных статьях, но на полноценную статью вроде не совсем тянут.

Примера пока что у меня два:
* Описание формата `cron` обзорно фигурирует в статье про GitHub Actions. Чтобы не дублировать, в статье про GitLab CI/CD я просто оставил ссылку не Википедию.
* В статьях про JS может быть полезно упомянуть, например, про алгоритм `SameValueZero`.

В обоих случаях хочется не дублировать даже такие небольшие абзацы, но и создавать отдельную страницу под них кажется оверхедом. И вообще, с ростом количества статей сложности и вопросы касательно навигации будут серьезнее. 

Что думаете?

**Comment by pepelsbey** [comment, marked as answer]
Привет! Мы обсудили редакцией эту идею и решили, что этот подход не совсем вписывается в жанр авторской документации. Это хорошо работает для кода, но такое компонентное деление на части не сработает для текста.

Нам кажется, что если какая-то часть про тот же cron попадается снова и снова, и важно её упоминать, то лучше написать статью об этом, даже если эта статья будет состоять из пяти абзацев. И дальше самым натуральным образом ссылаться на неё.
